packages/i18n/src/locales/en.i18n.json

```
@@ -3672,6 +3672,13 @@
   "Message_KeepHistory": "Keep Per Message Editing History",
   "Message_MaxAll": "Maximum Channel Size for ALL Message",
   "Message_MaxAllowedSize": "Maximum Allowed Characters Per Message",
+  "Message_ConfirmGroupMentions": "Ask for confirmation when using @all or @here mentions",
+  "Message_ConfirmGroupMentions_Description": "When enabled, a confirmation dialog will be shown before sending a message that mentions @all or @here in a channel with many members.",
+  "Message_ConfirmGroupMentions_MinMembers": "Minimum members to trigger @all/@here confirmation",
+  "Message_ConfirmGroupMentions_MinMembers_Description": "Only show the confirmation dialog if the room has at least this many members. Set to 0 to always confirm.",
+  "Group_mention_confirm_title": "Notify all members?",
+  "Group_mention_confirm_description": "This message will notify all {{count}} members of this channel. Are you sure you want to proceed?",
+  "Send_anyway": "Send anyway",
```

Biome (2.5.6)

[error] The key Send_anyway was already declared.

(lint/suspicious/noDuplicateObjectKeys)